app/src/main/kotlin/com/simplemobiletools/smsmessenger/helpers/ImageCompressor.kt

```diff
@@ -116,4 +130,55 @@ class ImageCompressor(private val context: Context) {
         }
     }
 
+    private inner class ResolutionConstraint(private val width: Int, private val height: Int) {
+
+        private fun decodeSampledBitmapFromFile(imageFile: File, reqWidth: Int, reqHeight: Int): Bitmap {
+            return BitmapFactory.Options().run {
+                inJustDecodeBounds = true
+                BitmapFactory.decodeFile(imageFile.absolutePath, this)
+
+                inSampleSize = calculateInSampleSize(this, reqWidth, reqHeight)
+
+                inJustDecodeBounds = false
+                BitmapFactory.decodeFile(imageFile.absolutePath, this)
+            }
+        }
+
+        private fun calculateInSampleSize(options: BitmapFactory.Options, reqWidth: Int, reqHeight: Int): Int {
+            // Raw height and width of image
+            val (height: Int, width: Int) = options.run { outHeight to outWidth }
```

Comment:
this looks pretty bad, write it in a readable form please

Reply:
As I said in description, I just C/P Compressor code here.
Is anything like that would please you ?:
```suggestion
            val height = options.outHeight
            val width = options.outWidth 
```